docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -360,7 +360,7 @@ Then elements located `$locator` are sorted by text in $sortingOrder order
 ==== *_Parameters_*
 
 . `$locator` - <<_locator>>
-. `$sortingOrder` - either https://www.computerhope.com/jargon/a/ascendin.htm[ASCENDING] or https://www.computerhope.com/jargon/d/descorde.htm[DESCENDING] order
+. `$sortingOrder` - either https://www.computerhope.com/jargon/a/ascendin.htm[ASCENDING] or https://www.computerhope.com/jargon/d/descorde.htm[DESCENDING] or CASE_INSENSITIVE_ASCENDING or CASE_INSENSITIVE_DESCENDING order
```

Comment:
please format the possible values in list-like view
also please provide possble user-friendly values like `case-insensitive ascending`